### PR TITLE
add 2.7 support

### PR DESCRIPTION
**Pull request by supernothing**

### CODE REVIEW

**Comment by Rizato** [comment]
Would it be good to re-enable tox and text in multiple envs?

**Comment by Rizato** [comment]
Tox is enabled, so we just have to run the 2.7 env too

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     author_email='info@polyswarm.io',
     url='https://github.com/polyswarm/polyswarm-artifact',
     license='MIT',
-    python_requires='>=3.5.2,<4',
+    python_requires='>=2.7,<4',
```
Comment (mrtizmoatwork) [inline setup.py]: Make sure to add a line to classifiers list like this:
        'Programming Language :: Python :: 2.7',

Comment (mrtizmoatwork) [inline setup.py]: also, by saying >=2.7 and <4, we are including 3.0-3.4. Do you know of any problems with 3.0-3.4 such that we do NOT want to support them?

Comment (supernothing) [reply setup.py]: I'm not aware of any such problems, that was intentional. Something funky with `enum`s in 2.7 though, still figuring it out.

Comment (Rizato) [reply setup.py]: Looks like they were added in 3.4

**Pull request closed without merging**